contracts/base/InterestConsensus.sol

```diff
@@ -47,29 +44,26 @@ contract InterestConsensus is Consensus, Initializable, InterestConsensusInterfa
         _;
     }
 
-    function initialize(
-        address lendersAddress,
-        uint256 initRequiredSubmissions,
-        uint256 initMaximumTolerance,
-        uint256 initResponseExpiry
-    ) public isNotInitialized() {
+    function initialize(address lendersAddress, address settingsAddress)
+        external
+        isNotInitialized()
+    {
         lendersAddress.requireNotEmpty("MUST_PROVIDE_LENDER_INFO");
-        require(initRequiredSubmissions > 0, "MUST_PROVIDE_REQUIRED_SUBS");
-        require(initResponseExpiry > 0, "MUST_PROVIDE_RESPONSE_EXP");
+        settingsAddress.requireNotEmpty("MUST_PROVIDE_SETTINGS");
 
-        initialize();
+        initialize(settingsAddress);
 
         lenders = lendersAddress;
-        requiredSubmissions = initRequiredSubmissions;
-        maximumTolerance = initMaximumTolerance;
-        responseExpiryLength = initResponseExpiry;
     }
 
     function processRequest(
         ZeroCollateralCommon.InterestRequest calldata request,
         ZeroCollateralCommon.InterestResponse[] calldata responses
     ) external isInitialized() isLenders() returns (uint256) {
```

Comment:
Shuold this be whenNotPaused()?

Reply:
This function is only called by Lenders.setAccruedInterest. That one contains the modifiers.